test/unit/messaging/process-data-message.test.js

```diff
@@ -0,0 +1,91 @@
+jest.mock('../../../app/messaging/validate-message')
+const { validateMessage: mockValidateMessage } = require('../../../app/messaging/validate-message')
+
+jest.mock('../../../app/cache')
+const { getCachedResponse: mockGetCachedResponse, setCachedResponse: mockSetCachedResponse, getCacheKey: mockGetCacheKey } = require('../../../app/cache')
+
+jest.mock('../../../app/messaging/send-message')
+const { sendMessage: mockSendMessage } = require('../../../app/messaging/send-message')
+
+jest.mock('../../../app/data')
+const { getData: mockGetData } = require('../../../app/data')
+
+const { cacheConfig, messageConfig } = require('../../../app/config')
+const { REQUEST_MESSAGE } = require('../../mocks/messaging/message')
+const { CATEGORY } = require('../../mocks/values/category')
+const { REQUEST_VALUE } = require('../../mocks/cache/request-value')
+const { REQUEST } = require('../../mocks/request')
+const { RESPONSE } = require('../../mocks/values/response')
+const { KEY } = require('../../mocks/cache/key')
+const { TYPE } = require('../../../app/constants/type')
+const { VALIDATION } = require('../../../app/constants/errors')
+
+const { processDataMessage } = require('../../../app/messaging/process-data-message')
```

Comment:
See my comment here: https://github.com/DEFRA/ffc-pay-data-hub/pull/1/files#r1165418243

Reply:
Same response :)

Reply:
I don't actually remember what I wrote here and the link doesn't seem to want to work for me to see what I was referencing - was it something related to grouping cacheConfig and messageConfig with processDataMessage?

Reply:
You asked if it should move up to join the others, but I said it would be best so it's clear it's the focus of the tests

Reply:
Yep, okay. I remember that now